resources/grafana/downstream/grafana-dashboard-dataplane-slos.yaml

```
@@ -169,7 +172,7 @@ data:
                 "uid": "${Datasource}"
               },
               "editorMode": "code",
-              "expr": "( sum(rate(cos_fleetshard_sync_connector_state_count_total{cos_connector_state=\"ready\",cluster_id=~\"$Cluster\"}[28d])) by (cluster_id) /\n( sum(rate(cos_fleetshard_sync_connector_state_count_total{cos_connector_state=\"ready\",cluster_id=~\"$Cluster\"}[28d])) by (cluster_id) + sum(rate(cos_fleetshard_sync_connector_state_count_total{cos_connector_state=\"failed_but_ready\",cluster_id=~\"$Cluster\"}[28d])) by (cluster_id) ) ) * 100",
+              "expr": "slo_connector_availability_success_rate:ratio_rate28d",
```

Dont you have to still include the cluster_id?
```suggestion
              "expr": "slo_connector_availability_success_rate:ratio_rate28d{cluster_id=~\"$Cluster\"}",
```

---

@lgarciaaco,
Added the "Cluster" variable back!